2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_setup.py:_flush():76] Configure stats pid to 11624
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_setup.py:_flush():76] Loading settings from C:\Users\DELL\.config\wandb\settings
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_setup.py:_flush():76] Loading settings from C:\Users\DELL\Downloads\FederatedLearning\OpenFL-extended\openfl-tutorials\boosting-examples\AdaBoostF_Heart_Failure\workspace\wandb\settings
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'openfl-tutorials\\boosting-examples\\AdaBoostF_Heart_Failure\\workspace\\AdaBoostF_Heart_Failure.py', 'program': 'AdaBoostF_Heart_Failure.py'}
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_init.py:_log_setup():507] Logging user logs to C:\Users\DELL\Downloads\FederatedLearning\OpenFL-extended\openfl-tutorials\boosting-examples\AdaBoostF_Heart_Failure\workspace\wandb\run-20230608_132846-dnu57j85\logs\debug.log
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_init.py:_log_setup():508] Logging internal logs to C:\Users\DELL\Downloads\FederatedLearning\OpenFL-extended\openfl-tutorials\boosting-examples\AdaBoostF_Heart_Failure\workspace\wandb\run-20230608_132846-dnu57j85\logs\debug-internal.log
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_init.py:init():547] calling init triggers
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_init.py:init():554] wandb.init called with sweep_config: {}
config: {}
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_init.py:init():595] starting backend
2023-06-08 13:28:46,676 INFO    MainThread:11624 [wandb_init.py:init():599] setting up manager
2023-06-08 13:28:46,708 INFO    MainThread:11624 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn, using: spawn
2023-06-08 13:28:46,723 INFO    MainThread:11624 [wandb_init.py:init():605] backend started and connected
2023-06-08 13:28:46,739 INFO    MainThread:11624 [wandb_init.py:init():695] updated telemetry
2023-06-08 13:28:47,286 INFO    MainThread:11624 [wandb_init.py:init():732] communicating run to backend with 60.0 second timeout
2023-06-08 13:29:47,894 ERROR   MainThread:11624 [wandb_init.py:init():758] encountered error: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2023-06-08 13:29:47,958 ERROR   MainThread:11624 [wandb_init.py:init():1163] Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
Traceback (most recent call last):
  File "C:\Users\DELL\AppData\Roaming\Python\Python38\site-packages\wandb\sdk\wandb_init.py", line 1145, in init
    run = wi.init()
  File "C:\Users\DELL\AppData\Roaming\Python\Python38\site-packages\wandb\sdk\wandb_init.py", line 764, in init
    raise error
wandb.errors.CommError: Run initialization has timed out after 60.0 sec. 
Please refer to the documentation for additional information: https://docs.wandb.ai/guides/track/tracking-faq#initstarterror-error-communicating-with-wandb-process-
2023-06-08 13:29:49,007 WARNING MsgRouterThr:11624 [router.py:message_loop():77] message_loop has been closed
